bin/cli.js

```diff
@@ -288,20 +288,33 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				}).apply(compiler);
 			}
 			if (outputOptions.infoVerbosity === "verbose") {
-				if (argv.w) {
-					compiler.hooks.watchRun.tap("WebpackInfo", compilation => {
+				if (compiler.compilers) {
+					compiler.compilers.forEach(comp => {
+						setUpHookForCompilation(comp, argv);
+					});
+				} else {
+					setUpHookForCompilation(compiler, argv);
+				}
+			}
```

Comment:
Can be writter:
```js
const compilers = compiler.compilers ? compiler.compilers : [compiler];

compilers.map((compiler) =>setUpHookForCompilation(compiler, argv));
```

Reply:
I think  It's the shorthand syntax for watch. Actually it was previously used there. So I just used it :smile:

Reply:
I am sorry. I am just not so good with github. Accidentally, I replied in this which i should not. my bad...  :hand: